ui/admin/app/components/worker-filter/index.hbs

```diff
@@ -34,6 +34,47 @@
     {{/if}}
   </Hds::Form::Field>
 
+  <Hds::Separator />
+
+  <Hds::Form::Toggle::Field
+    name='show_filter_generator'
+    {{on 'change' this.toggleFilterGenerator}}
+    as |F|
+  >
+    <F.Label>{{t 'worker-filter-generator.toggle.title'}}</F.Label>
+    <F.HelperText>{{t
+        'worker-filter-generator.toggle.description'
+      }}</F.HelperText>
+  </Hds::Form::Toggle::Field>
+
+  <Hds::Separator />
+
+  {{#if this.showFilterGenerator}}
+    <Hds::Form::Radio::Group @name='filter_generator' as |G|>
+      <G.Legend>{{t
+          'worker-filter-generator.filter-generator.title'
+        }}</G.Legend>
+      <G.HelperText>
+        {{t 'worker-filter-generator.filter-generator.description'}}
+        <br />
+        <Hds::Link::Inline
+          @href={{doc-url 'target.worker-filters'}}
```

Comment:
Can we rename this doc-url to not be target specific?

Reply:
great suggestion, updated to be `worker-filters-format`. Does that work or do u have a diff. name suggestion?

Reply:
I like that name. 👍🏼